testing/bats/kubernetes/standalone.bats

```diff
@@ -63,6 +63,28 @@ createDefaultDeployment() {
     sleep 30
 }
 
+# Aim to use locals in case we want to parallelise to prevent overwriting globals
+setupPortForwarding() {
+    # Port forward into the K8S cluster
+    local PORT_FORWARD_PID_FILE=$1
+    kubectl -n "$TEST_NAMESPACE" port-forward svc/couchbase-grafana-http "$CMOS_PORT:8080" &
+    echo "$!" > "${PORT_FORWARD_PID_FILE}"
+
+    # Takes a little while to actually set up
+    local LOCAL_SERVICE_URL="localhost:$CMOS_PORT"
+    local ATTEMPTS=0
+    local MAX_ATTEMPTS=6
+    until curl -s -o /dev/null "$LOCAL_SERVICE_URL"; do
+        # shellcheck disable=SC2086
+        if [[ $ATTEMPTS -gt $MAX_ATTEMPTS ]]; then
+            fail "unable to communicate with CMOS on $LOCAL_SERVICE_URL after $ATTEMPTS attempts"
+        fi
+        ATTEMPTS=$((ATTEMPTS+1))
+        echo "Attempt $ATTEMPTS of $MAX_ATTEMPTS for CMOS on $LOCAL_SERVICE_URL"
+        sleep 10
+    done
```

Comment:
We should be using the helper here but I think it was introduced in a different branch - if so, no big deal, we can sort it later

Reply:
Yeah I think that's on yours so probably need to resolve the conflicts once in.